packages/widget-markdown/tests/Widget.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import Widget from '../src'
+import { getEventListener, GlobalProvider } from '@vscode-marquee/utils'
+import { MarkdownDocument, State } from '../src/types'
+import { WIDGET_ID } from '../src/Context'
+
+const widgetChannel = getEventListener<State>(WIDGET_ID)
+
+const { component: Component } = Widget
+
+const mockFileList = (fileList: MarkdownDocument[]) => {
+  widgetChannel.broadcast({
+    markdownDocuments: fileList,
+    markdownDocumentSelected: undefined,
+  })
```

Comment:
@christian-bromann I am trying to simulate and update from the backend, but this doesn't seem to do it. 
What's the proper way of doing this without having to mock?